src/components/button.js

```diff
@@ -2,16 +2,27 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = props => {
-  const { name, color, wide } = props;
+  const {
+    name, color, wide, clickHandler,
+  } = props;
+  const handleClick = buttonName => clickHandler(buttonName);
   return (
-    <button style={{ backgroundColor: color, width: wide ? '50%' : '25%' }} className="button" type="button">{name}</button>
+    <button
+      style={{ backgroundColor: color, width: wide ? '50%' : '25%' }}
+      className="button"
+      type="button"
+      onClick={() => { handleClick(name); }}
```

Comment:
You could simplify it a bit if you attach the `hadleClick` to the `onClick` event, and then modify your `handleClick` function to use `name` instead of the argument, something like this:

```js
const Button = ({
  name, color, wide, clickHandler,
}) => {
  const handleClick = () => clickHandler(name);
  return (
    <button
      style={{ backgroundColor: color, width: wide ? '50%' : '25%' }}
      className="button"
      type="button"
      onClick={handleClick}
    >
      {name}
    </button>
  );
};
```